src/Compilers/CSharp/Portable/Parser/Lexer.cs

```diff
@@ -1815,7 +1808,7 @@ private bool ScanIdentifier_SlowPath(ref TokenInfo info)
                                 // Debugger syntax allows @0x[hexdigit]+ for object address identifiers.
                                 if (info.IsVerbatim &&
                                     this.ModeIs(LexerMode.DebuggerSyntax) &&
-                                    (char.ToLower(TextWindow.PeekChar(1)) == 'x'))
+                                    (char.ToLowerInvariant(TextWindow.PeekChar(1)) == 'x'))
```

Comment:
This surely must have been a mistake? I can't imagine a scenario where parsing would depend on the current culture.